### ISSUE TITLE
JSON output flag for using programatically

### ISSUE THREAD

**Issue by danielloader** [Tantalor93/dnspyre#93]
Having used dnspyre for a while it's been great, but I kept ended up either grepping the stdout output or using --csv flag going to a third or fourth pipe to pass it downstream to do something with it. However the csv flag doesn't give me the rest of the output summary, like successful vs failed request counts and the p99 etc overviews. 

Would it be possible to make a mode where the stdout output is json only, and non json output goes into stderr? 

I was considering running dnspyre in a loosely isolated distributive mode on kubernetes but need some way to take the response from a worker pod back to a controlling operator, and json probably makes the most sense - can capture the stdout and curl it back to the operator http server to collect results per worker.

**Comment by Tantalor93** [comment]
Hello, thanks for opening the issue!

So If I understand it correctly, you would appreciate a new option `- -json`, which when specified will change the report output to be printed as json, something like
```
 {
    “totalRequests”:20,
    “totalErrors” : 1,
     “responses” :  {
        “NOERROR” : 17,
         “SERVFAIL” : 1
     },
     “qps”: 5,
     “benchmarkDurationInSeconds” : 30
 }
```

I will think about the JSON structure, but it makes sense 👍

**Comment by danielloader** [comment]
Yeah looks good, the granularity of the csv ranges would be good in an array of objects or similar. That way I can count how many are over an arbitrary maximum response time limit too.

**Comment by Tantalor93** [comment]
Hello @danielloader , I have released a new version `2.8.0`, which adds support for a new `--json` flag, which will output results in a JSON format. Please check whether it fits your use case, adjustments to format can be made :)
